16/umbraco-cms/customizing/property-editors/property-value-converters.md

```diff
@@ -102,31 +131,23 @@ The property value will be cached until its _element_ is modified. The element i
 * For properties used at the page level, the element is the entire page.
 * For properties contained within Block List items, the element is the individual Block List item.
 
-This is the most commonly used cache level and should be your default, unless you have specific reasons to do otherwise.
```

Comment:
I'm not sure why this was removed from here and inserted above. I think it does belong here (see also my other comment above).